src/test/unit_tests/test_s3_ops.js

```diff
@@ -423,6 +477,68 @@ mocha.describe('s3_ops', function() {
             const res2 = await s3.listObjects({ Bucket: bucket_name }).promise();
             assert.strictEqual(res2.Contents.length, (res1.Contents.length + 2));
         });
+
+        mocha.it('should copy text-file tagging', async function() {
+            if (is_azure_mock) this.skip();
+            this.timeout(120000);
+            const params = {
+                Bucket: bucket_name,
+                Key: text_file1,
+                Tagging: {
+                    TagSet: [{
+                        Key: 's3ops',
+                        Value: 'copy_tagging'
+                    }]
+                }
+            };
+
+            let notSupported = false;
+            await s3.putObjectTagging(params).on('error', err => {
```

Comment:
does every error here mean putObjectTagging is not supported? maybe we should be more specific?

Reply:
I have added the error. I have to mention that I never actually saw this error happens. I just saw that whenever putObjectTagging is called in the code this error is checked. whoever added it (Sanjeev Gupta) doesn't work in the company anymore. I added it to be consistent with the code, in case there are situations I have missed

Reply:
I don't think we should follow his logic, especially if we don't understand it. If you have some cases this test needs to be skipped - keep it, but otherwise, I think it's better to just fail on any error thrown (avoiding this on error/complete).

Reply:
I removed the the error checking